lib/config.js

```diff
@@ -188,15 +186,12 @@ export default class Config {
   parseAppJs (root) {
     let res = {}
     const filename = path.join(root, 'app.js')
+    require('babel-core/register')
     try {
-      const contents = transformFileSync(filename, {
-        filename: filename,
-        presets: [require('babel-preset-es2015'), require('babel-preset-stage-2')]
-      }).code
-      const mod = _eval(contents)
+      const mod = require(filename)
       res = mod.__esModule ? mod.default : mod
     } catch (err) {
-      if (err.code !== 'ENOENT') { throw new Error(err) }
+      if (err.code !== 'MODULE_NOT_FOUND') { throw new Error(err) }
```

Comment:
Hmmm, well we can run an `existsSync` check first instead then it wouldn't surpress it?